exchanges/lbank/lbank.go

```diff
@@ -0,0 +1,542 @@
+}
+
+// QueryOrder finds out information about orders
+func (l *Lbank) QueryOrder(pair, orderIDs string) (QueryOrderResponse, error) {
+	var resp QueryOrderResponse
+	params := url.Values{}
+	params.Set("symbol", pair)
+	params.Set("order_id", orderIDs)
+	path := fmt.Sprintf("%s/v%s/%s?", l.APIUrl, lbankAPIVersion, lbankQueryOrder)
+	return resp, l.SendAuthHTTPRequest("POST", path, params, &resp)
+}
+
+// QueryOrderHistory finds order info in the past 2 days
+func (l *Lbank) QueryOrderHistory(pair, pageNumber, pageLength string) (OrderHistoryResponse, error) {
+	var resp OrderHistory
+	params := url.Values{}
+	params.Set("symbol", pair)
+	params.Set("current_page", pageNumber)
+	params.Set("page_length", pageLength)
+	path := fmt.Sprintf("%s/v%s/%s?", l.APIUrl, lbankAPIVersion, lbankQueryHistoryOrder)
+	log.Println(path)
+	err := l.SendAuthHTTPRequest("POST", path, params, &resp)
+	if err != nil {
+		return OrderHistoryResponse{}, err
+	}
+
+	var rt OrderHistoryResponse
+	rt.CurrentPage = resp.CurrentPage
+	rt.ErrorCode = resp.ErrorCode
+	rt.PageLength = resp.PageLength
+	rt.Result = resp.Result
+	rt.Total = resp.Total
+
+	var orders []OrderResponse
+	err = json.Unmarshal(resp.Orders, &orders)
+	if err == nil {
+		rt.Orders = orders
+		return rt, nil
+	}
+
+	var order OrderResponse
+	err = json.Unmarshal(resp.Orders, &order)
+	if err == nil {
+		rt.Orders = append(rt.Orders, order)
+		return rt, nil
+	}
+
+	return rt, nil
+}
+
+// GetPairInfo finds information about all trading pairs
+func (l *Lbank) GetPairInfo() ([]PairInfoResponse, error) {
+	var resp []PairInfoResponse
+	path := fmt.Sprintf("%s/v%s/%s?", lbankAPIURL, lbankAPIVersion, lbankPairInfo)
+	return resp, l.SendHTTPRequest(path, &resp)
+}
+
+// GetOpenOrders gets opening orders
+func (l *Lbank) GetOpenOrders(pair string, pageNumber, pageLength int64) (OpenOrderResponse, error) {
+	var resp OpenOrderResponse
+	params := url.Values{}
+	params.Set("symbol", pair)
+	params.Set("current_page", strconv.FormatInt(pageNumber, 10))
+	params.Set("page_length", strconv.FormatInt(pageLength, 10))
+	path := fmt.Sprintf("%s/v%s/%s", l.APIUrl, lbankAPIVersion, lbankOpeningOrders)
+	return resp, l.SendAuthHTTPRequest("POST", path, params, &resp)
+}
+
+// USD2RMBRate finds USD-CNY Rate
+func (l *Lbank) USD2RMBRate() (ExchangeRateResponse, error) {
+	var resp ExchangeRateResponse
+	path := fmt.Sprintf("%s/v%s/%s", lbankAPIURL, lbankAPIVersion, lbankUSD2CNYRate)
+	return resp, l.SendHTTPRequest(path, &resp)
+}
+
+// GetWithdrawConfig gets information about withdrawals
+func (l *Lbank) GetWithdrawConfig(assetCode string) ([]WithdrawConfigResponse, error) {
+	l.Verbose = true
+	var resp []WithdrawConfigResponse
+	params := url.Values{}
+	if assetCode != "" {
+		params.Set("assetCode", assetCode)
+	}
+	path := fmt.Sprintf("%s/v%s/%s?%s", lbankAPIURL, lbankAPIVersion, lbankWithdrawConfig, params.Encode())
+	return resp, l.SendHTTPRequest(path, &resp)
+}
+
+// Withdraw withdraws
```

Comment:
Please update this. While an accurate description, something more like "sends a withdrawal request" is better since there can be errors returned